### PR TITLE
HPCC-15277 Min/Max Skew Renamed

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-15277

Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@jakesmith Please review

File: esp/src/eclwatch/ESPGraph.js [hunk 1]
```diff
@@ -505,6 +505,8 @@ define([
                                 }
                                 if (edge.maxskew && edge.minskew) {
                                     edge._eclwatchSkew = "+" + edge.maxskew + "%%, -" + edge.minskew + "%%";
+                                } else if (edge.SkewMaxRowsProcessed && edge.SkewMinRowsProcessed) {
+                                    edge._eclwatchSkew = "+" + edge.SkewMaxRowsProcessed + ", " + edge.SkewMinRowsProcessed;
```
Comment (jakesmith) [inline esp/src/eclwatch/ESPGraph.js]: @GordonSmith 
Do we need to support both? i.e. do we needs lines on 506+507 ?

Comment (GordonSmith) [inline esp/src/eclwatch/ESPGraph.js]: I suspect not here - in the GraphControl we would for sure.  I will update...

**Comment by GordonSmith** [comment]
@richardkchapman @jakesmith Fixed

**Comment by jakesmith** [comment]
@richardkchapman - looks good to merge

**Pull request merged**